.github/workflows/CI_PR_merge_checks.yml

```diff
@@ -25,3 +25,22 @@ jobs:
         run: yarn lint
       - name: Test
         run: yarn test
+      - name: Validate generate-mjml-react.ts has been run
+        run: |
```

Comment:
Can this be moved to a .sh file instead of being all define here inside the yml? Ideally this would just read as `./scripts/validateMjmlReactHasBeenBuilt.sh` or something similar